olive/model.py

```diff
@@ -234,6 +237,52 @@ def prepare_session(self, inference_settings: Dict[str, Any], device: Device, ra
 
         return get_ort_inference_session(self.model_path, inference_settings)
 
+    def nodes(self):
+        all_nodes = []
+        for graph in self.graphs():
+            for node in graph.node:
+                all_nodes.append(node)
+        return all_nodes
+
+    def graph(self):
+        return self.load_model().graph
+
+    def graphs(self):
+        from onnx import AttributeProto, GraphProto
```

Comment:
>from onnx import AttributeProto, GraphProto [](http://example.com/codeflow?start=8&length=43)

could we move the onnx import to the top of the file it seems the onnx module is already imported.

Reply:
Ah I didn't notice that. I'll move it to the top.